test/lsp/utils/text_edit.vimspec

```diff
@@ -548,7 +548,7 @@ Describe lsp#utils#text_edit
                     \ }])
 
             let l:buffer_text = s:get_text()
-            Assert Equals(l:buffer_text, ['x', 'y', 'z', 'b', ''])
```

Comment:
IMO, The buffer is `['foo', 'b']` and range-end is `line: 1, char: 1`.
So `b` character is in range to replacement.